3.6/aql/operators.md

```diff
@@ -8,6 +8,8 @@ Operators
 AQL supports a number of operators that can be used in expressions.  There are
 comparison, logical, arithmetic, and the ternary operator.
 
+Fire up your database and try out the power of AQL in just a few clicks with ArangoDB Oasis: the Cloud Service for ArangoDB. [Start your free 14-day trial here](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=top_pages&utm_campaign=docs_traffic){:target="_blank"}.
```

Comment:
This page isn't suitable for an Oasis plug IMO. No one tries out comparison operators, they are useless on their own.

Reply:
Let's keep the text and move it to the bottom

Reply:
Moved to end of page as discussed.